### PR TITLE
Linux cleanup

### PR DESCRIPTION
**Pull request by corporateshark**

### CODE REVIEW

File: src/panel.cpp [hunk 1]
```diff
@@ -494,13 +494,13 @@ static int* CheckMode( int* m )
 			break;
 
 		default:
-			*m = PanelWin::MEDIUM;
+			m = PanelWin::MEDIUM;
```
Comment (corporateshark) [inline src/panel.cpp]: Why not just ``return PanelWin::MEDIUM``?

File: src/panel.cpp [hunk 2]
```diff
@@ -513,7 +513,8 @@ PanelWin::PanelWin( Win* parent, int* mode )
 	_current( 0 ),
 	_viewMode( CheckMode( mode ) ), //MEDIUM),
```
Comment (corporateshark) [inline src/panel.cpp]: Are you sure by this time we have a valid (loaded) value of ``mode``? Probably, this was the whole point of having a pointer.

Comment (singalen) [inline src/panel.cpp]: Point taken, will review it.

Comment (singalen) [inline src/panel.cpp]: Now I'm pretty sure. These are the proof:
* `g_WcmConfig.Load()` is invoked in `nc.cpp` before any UI;
* `mode` is a constructor argument. If it was uninitialized before, then the whole structure is pretty much screwed;
* I've tested my version by hand.

**Pull request merged**